modules/core/src/main/java/org/apache/ignite/cache/CacheMetrics.java

```diff
@@ -243,13 +243,27 @@
      */
     public int getSize();
 
+    /**
+     * Gets number of non-{@code null} values in the cache.
+     *
+     * @return Number of non-{@code null} values in the cache.
+     */
+    public long getSizeLong();
```

Comment:
Maybe it makes sense to describe difference with similar getSize() method?

Reply:
Done